app/models.py

```
@@ -76,23 +287,76 @@ class BotRequest(BaseModel):
+    )
+    llm_model: str | None = Field(
+        None,
+        min_length=1,
+        max_length=120,
+        description="Provider model for this bot. Defaults to provider-specific env vars.",
+    )
+    speech_speed: float | None = Field(
+        None,
+        ge=0.6,
+        le=1.5,
+        description="TTS speaking speed multiplier. Honored range is 0.6–1.5 (the runner clamps to it); values outside are rejected. Defaults to CARTESIA_TTS_SPEED, TTS_SPEED, SPEECH_SPEED, or the runner default.",
```

Ruff (0.15.21)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)